### ISSUE TITLE
[Android] lineHeight on Android has problem on first line

### ISSUE THREAD

**Issue by yangnana11** [facebook/react-native#20644]
Environment:
  OS: macOS High Sierra 10.13.5
  Node: 8.9.1
  Yarn: 1.6.0
  npm: 6.2.0
  Watchman: 4.9.0
  Xcode: Xcode 9.4.1 Build version 9F2000
  Android Studio: 3.1 AI-173.4819257

Packages: (wanted => installed)
  react: 16.3.1 => 16.3.1
  react-native: 0.55.1 => 0.55.1

There is a issue about this, but it is already closed and I can't reopen it again, so I start a new one ...
I made a Korean app, and the lineHeight not work well in Android, the first line is wrong. I applied lineHeight cause it is too crowd and hard to read.
This is the result:

![image](https://user-images.githubusercontent.com/39118779/44014234-aba62964-9f05-11e8-95e6-85cc1c213b33.png)

The style:
```
textContent: {
    fontFamily: Fonts.type.notoRegular,
    fontSize: 16,
    color: Colors.detailText,
    ...Platform.select({
      ios: {
        textAlign: 'center'
      },
      android: {
        lineHeight: 25
      }
    })
  }
```

And the code:
```
<Text style={styles.textContent}>
            블록체인은 디지털 경제 시대의 가속도를 붙이는 코어기술이다. 한 기업에 집중됐던 정보가 원래의 주인인 개인들에 분산됨으로써 이들이 직접 자신의 정보를 소유하고 관리할 수 있게 해준다. 4차 산업혁명의 주요 기술인 블록체인 기술을 기반으로 한 다양한 플랫폼 개발이 활발히 진행되고 있어 관심을 끈다.
            블록체인 운영자문업체 넥스트블록(대표 정규화)은 전세계 블록체인 프로젝트의 암호화폐공개(ICO, Initial Coin Offering)를 보다 편하게 참여할 수 있는 플랫폼인 넥스트아이씨오(NextICO)를 최근 오픈해 부동산 거래 및 자금 조달 플랫폼인 비홈(BHOM)을 비롯한 다양한 ICO 프로젝트들을 판매하고 있다.
            //go on
</Text>
```
Some one said I could apply
```
{'\n'}
```
at the beginning to avoid the first line, but it doesn't work..

I hope for any suggestion.
Thank you very much.

**Comment by yangnana11** [comment]
Hello,

Thank you very much for your response. After I upgrade 0.56, there are a lot of errors which I don't know where should I start. Is there any requirement for 0.56? This is the original package.json file (before I upgrade to 0.56)

```
{
  "name": "app",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "start": "node node_modules/react-native/local-cli/cli.js start",
    "test": "jest",
    "clean": "rm -rf $TMPDIR/react-* && watchman watch-del-all && npm cache clean --force",
    "clean:android": "cd android/ && ./gradlew clean && cd .. && react-native run-android",
    "newclear": "rm -rf $TMPDIR/react-* && watchman watch-del-all && rm -rf ios/build && rm -rf node_modules/ && npm cache clean --force && npm i",
    "test:watch": "jest --watch",
    "updateSnapshot": "jest --updateSnapshot",
    "coverage": "jest --coverage && open coverage/lcov-report/index.html || xdg-open coverage/lcov-report/index.html",
    "android:build": "cd android && ./gradlew assembleRelease",
    "android:install": "cd android && ./gradlew assembleRelease && ./gradlew installRelease",
    "android:hockeyapp": "cd android && ./gradlew assembleRelease && puck -submit=auto app/build/outputs/apk/app-release.apk",
    "android:devices": "$ANDROID_HOME/platform-tools/adb devices",
    "android:logcat": "$ANDROID_HOME/platform-tools/adb logcat *:S ReactNative:V ReactNativeJS:V",
    "android:shake": "$ANDROID_HOME/platform-tools/adb devices | grep '\\t' | awk '{print $1}' | sed 's/\\s//g' | xargs -I {} $ANDROID_HOME/platform-tools/adb -s {} shell input keyevent 82",
    "storybook": "storybook start -p 7007",
    "lint": "standard --verbose | snazzy",
    "lintdiff": "git diff --name-only --cached --relative | grep '\\.js$' | xargs standard | snazzy",
    "fixcode": "standard --fix",
    "git-hook": "npm run lint -s && npm run test -s"
  },
  "dependencies": {
    "apisauce": "^0.14.2",
    "format-json": "^1.0.3",
    "identity-obj-proxy": "^3.0.0",
    "lodash": "^4.17.4",
    "native-base": "^2.7.2",
    "prop-types": "15.6.0",
    "querystringify": "1.0.0",
    "ramda": "^0.25.0",
    "react": "16.3.1",
    "react-native": "0.55.1",
    "react-native-animatable": "^1.3.0",
    "react-native-config": "^0.10.0",
    "react-native-device-info": "0.11.0",
    "react-native-swiper": "^1.5.13",
    "react-native-vector-icons": "^5.0.0",
    "react-navigation": "1.0.0-beta.11",
    "react-navigation-redux-helpers": "^1.0.4",
    "react-redux": "^5.0.6",
    "redux": "^3.7.2",
    "redux-saga": "^0.16.0",
    "reduxsauce": "0.7.0",
    "seamless-immutable": "^7.1.2"
  },
  "devDependencies": {
    "@storybook/addon-storyshots": "^3.4.0",
    "@storybook/addons": "^3.4.0",
    "@storybook/channels": "^3.4.0",
    "@storybook/react-native": "^3.4.0",
    "babel-core": "6.26.0",
    "babel-eslint": "7.1.1",
    "babel-jest": "23.4.2",
    "babel-plugin-ignite-ignore-reactotron": "^0.3.0",
    "babel-preset-env": "^1.6.1",
    "babel-preset-react-native": "^3.0.2",
    "enzyme": "^3.2.0",
    "enzyme-adapter-react-16": "^1.1.0",
    "husky": "^0.14.3",
    "ignite-animatable": "^1.0.0",
    "ignite-dev-screens": "^2.3.0",
    "ignite-ir-boilerplate-andross": "^2.3.3",
    "ignite-standard": "^1.0.0",
    "ignite-vector-icons": "^1.1.0",
    "jest": "23.4.2",
    "mockery": "^2.1.0",
    "react-dom": "16.3.0",
    "react-test-renderer": "16.3.1",
    "reactotron-react-native": "^2.0.0-alpha.3",
    "reactotron-redux": "^2.0.0-alpha.3",
    "reactotron-redux-saga": "^2.0.0-alpha.3",
    "snazzy": "^7.1.1",
    "standard": "10.0.2"
  },
  "jest": {
    "testMatch": [
      "<rootDir>/Tests/**/*.js",
      "**/?(*.)(spec|test).js?(x)"
    ],
    "testPathIgnorePatterns": [
      "/node_modules/",
      "<rootDir>/Tests/Setup.js"
    ],
    "moduleNameMapper": {
      "^.+\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "identity-obj-proxy"
    },
    "setupFiles": [
      "<rootDir>/Tests/Setup"
    ],
    "preset": "react-native"
  },
  "config": {},
  "standard": {
    "parser": "babel-eslint",
    "globals": [
      "describe",
      "test",
      "jest",
      "expect",
      "fetch",
      "navigator",
      "__DEV__",
      "XMLHttpRequest",
      "FormData",
      "React$Element"
    ]
  }
}
```

**Comment by yangnana11** [comment]
Thanks

**Comment by isabel-thx** [comment]
Why is this issue closed? I'm still facing this problem in RN 0.59.8. Was it fixed then reverted??

**Comment by jarredt** [comment]
Having this issue as well on 0.59.8.
